docs/contourplot/posfuns/bottom.points.html

```diff
@@ -65,13 +65,16 @@ <h1><a href="../../index.html">directlabels</a> - contourplot -
 
 <center>
 <table>
-<tr><td>Please contact <a href="http://cbio.ensmp.fr/~thocking/">Toby
-Dylan Hocking</a> if you are using <a href="http://directlabels.r-forge.r-project.org/">directlabels</a> or have ideas to
-contribute, thanks!</td>
+<tr><td>Please
+contact <a href="http://tdhock.github.io/">Toby
+Dylan Hocking</a> if you are
+using <a href="https://tdhock.github.io/directlabels/">directlabels</a>
+or have ideas to contribute, thanks!</td>
 </tr>
-<tr><td align="center">Documentation website generated from comments
-in R source code
-using <a href="http://inlinedocs.r-forge.r-project.org">inlinedocs</a>.</td>
+<tr><td align="center">Documentation website generated from source
+code version 2014.1.27
+(svn revision 675)
```

Comment:
this should be changed eventually, is now a good time?

Reply:
Yep I'll do it now, thanks for pointing that out!

Reply:
Do we need to include the git version info? The current code (lines [75](https://github.com/tdhock/directlabels/blob/8f8a4f27c9eb0d207dd3a01a0d58fae31496753b/R/doc.R#L75) to [77](https://github.com/tdhock/directlabels/blob/8f8a4f27c9eb0d207dd3a01a0d58fae31496753b/R/doc.R#L77)) provides the following information:
![image](https://user-images.githubusercontent.com/30123691/122364437-62c3f780-cf77-11eb-8f4e-d61975c38fbc.png)